recipe/patches/0004-test-allocator.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/absl/container/CMakeLists.txt b/absl/container/CMakeLists.txt
+index 449a2cad17e..ee9ca9c3c4a 100644
```

Comment:
Please generate this patch from https://github.com/abseil/abseil-cpp/commit/779a3565ac6c5b69dd1ab9183e500a27633117d5 using `git format-patch`.

If you want, here are more complete instructions:
```
git clone https://github.com/abseil/abseil-cpp
cd abseil-cpp
git checkout tags/20240116.2
find ../path/to/abseil-cpp-feedstock/recipe/patches/ | xargs git am      # without your 0004-* in recipe/patches
git cherry-pick 779a3565ac6c5b69dd1ab9183e500a27633117d5
git format-patch tags/20240116.2 --no-signature
```

Reply:
Thanks for the instructions. They were very helpful, and I have updated the patch as requested.